.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.20.2
```

Comment:
I'm not quite sure what the purpose of this Node.js version declaration is here?

Reply:
Manage multiple runtime versions by
https://asdf-vm.com/

Reply:
COOL, it looks good, but I've already set the .nvmrc to change the Node.js version automatically, and `v16.20.2` is a specific version number. Most of us might not necessarily have this exact version. For example, mine is `v16.14.2`

Reply:
Or let's do this: ignore this modification for now and use it locally. Because I currently don't want to add duplicate configuration files to the project, I need to maintain consistency.

I'll look at this feature to see if we need to switch to this tool. If it works, we'll add it back for everyone to use. After the modification, I'll approve this PR. Thank you very much for your contribution.

Reply:
`.tool-versions` reverted  and ignored.